keyboards/custommk/elysian/keyboard.json

```diff
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "customMK",
+    "keyboard_name": "Elysian",
+    "maintainer": "customMK",
+    "bootloader": "stm32-dfu",
```

Comment:
```suggestion
    "bootloader": "stm32-dfu",
    "debounce": 10,
```

Reply:
Moved debounce to data driven, thank you!